test/test_von_mises.cpp

```diff
@@ -330,11 +331,29 @@ void test_spots(RealType)
 
   // Error tests:
   check_out_of_range<boost::math::von_mises_distribution<RealType> >(0, 1); // (All) valid constructor parameter values.
-    
+
   BOOST_MATH_CHECK_THROW(quantile(von_mises_distribution<RealType>(0, 1), -1), std::domain_error);
   BOOST_MATH_CHECK_THROW(quantile(von_mises_distribution<RealType>(0, 1), 2), std::domain_error);
 } // template <class RealType>void test_spots(RealType)
 
+template <typename RealType>
+void test_symmetry(RealType)
+{
+	RealType const pi = boost::math::constants::pi<RealType>();
+	RealType delta = 1.0 / (1 << 4);
+  for (RealType mean = 0; mean < pi; mean += delta) {
+    for (RealType conc = 0; conc < 100; conc = (conc + 1) * 1.5 - 1) {
+      von_mises_distribution<RealType> dist(mean, conc);
+      for (RealType x = 0; x < pi; x += delta) {
+        CHECK_ULP_CLOSE(pdf(dist, mean + x),
+                        pdf(dist, mean - x), 2);
```

Comment:
Do you get 2 ULP accuracy in long double as well?

Reply:
Yes.